spec/api/holdings_api_spec.rb

```diff
@@ -232,8 +232,62 @@ def parse_body(call)
         build(:holding, organization: "umich", ocn: ocn),
         build(:holding, organization: "smu", ocn: ocn)
       )
-      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1, organization: "umich")))
+      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1)))
       expect(response["organizations"].sort).to eq ["smu", "umich"]
     end
   end
+
+  describe "v1/item_held_by_brlm" do
+    it "returns an application error if the item_id does not match an ht_item" do
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
```

Comment:
Would it make sense to further extract `parse_body(v1(url_template(` to de-duplicate that in the tests?

Reply:
simplified helper methods somewhat